core/field_dropdown.js

```diff
@@ -465,20 +462,16 @@ Blockly.FieldDropdown.prototype.render_ = function() {
  * @private
  */
 Blockly.FieldDropdown.prototype.renderSelectedImage_ = function() {
-  // Image option is selected.
-  this.imageElement_ = Blockly.utils.createSvgElement('image',
-      {
-        'y': 5,
-        'height': this.imageJson_.height + 'px',
```

Comment:
Where is the height of the image element getting set now?  Does it ever change?  (I assume width is set/modified in the same places.)

Reply:
I didn't think there was a need to set the height and width of the image element. It looks like it handles different sizes just fine:

![DropdownSizeExample](https://user-images.githubusercontent.com/25440652/58661693-58c1f400-82dd-11e9-8b39-177e28771131.gif)

But I checked the[ docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image#Specific_attributes) since you mentioned it, and they do say a height and width is required, so it's probably just a gap in my knowledge.

Readded setting height and width for safety.